guides/common/modules/con_managing-ansible-content.adoc

```diff
@@ -0,0 +1,6 @@
+[id="Managing_Ansible_Content_{context}"]
+= Managing Ansible Content
+
+You can import Ansible collections from several sources to {ProjectServer}.
```

Comment:
```suggestion
You can import Ansible collections from multiple sources to {ProjectServer}.
```
Not sure about this. Consider optional

Reply:
Vale for RH actually flags `multiple` as a word that's too complex.